From 5343dc569ab683fda6a9874919f9141aa90f00c0 Mon Sep 17 00:00:00 2001
From: lodriguez <github@bremsspur.org>
Date: Fri, 23 May 2025 15:26:57 +0200
Subject: [PATCH] remove OSMesa

OSMesa is depricated for quite a while and got removed with Mesa 25.1.0
---
 src/slic3r/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/slic3r/CMakeLists.txt b/src/slic3r/CMakeLists.txt
index b264030d764..ac2697c24b0 100644
--- a/src/slic3r/CMakeLists.txt
+++ b/src/slic3r/CMakeLists.txt
@@ -634,9 +634,9 @@ elseif (CMAKE_SYSTEM_NAME STREQUAL "Linux")
     FIND_LIBRARY(WAYLAND_EGL_LIBRARIES    NAMES wayland-egl)
     FIND_LIBRARY(WAYLAND_CLIENT_LIBRARIES NAMES wayland-client)
     find_package(CURL REQUIRED)
-    target_link_libraries(libslic3r_gui ${DBUS_LIBRARIES} OSMesa)
     target_link_libraries(libslic3r_gui
         OpenGL::EGL
+        ${DBUS_LIBRARIES}
         ${WAYLAND_SERVER_LIBRARIES}
         ${WAYLAND_EGL_LIBRARIES}
         ${WAYLAND_CLIENT_LIBRARIES}
